Shared/AppConstants.swift

```diff
@@ -4,6 +4,11 @@
 
 import UIKit
 
+public struct InternalScheme {
+    public static let scheme = "internal"
+    public static let url = "\(scheme)://local"
+}
```

Comment:
We use these constants everywhere.

Reply:
nit: I wonder if it makes sense to hang these off of `URL` somehow?